### ISSUE TITLE
update file `php-function-names-933151.data`

### ISSUE THREAD

**Issue by fzipi** [coreruleset/coreruleset#2685]
These must be addressed as a group:
- #2684
- #2685
- #3021
- #3022 
 
## Rule 933151

~1300 words of lesser importance. This includes most PHP functions and keywords.
```
Examples: 'addslashes', 'array_diff'.
```
For performance reasons, the @pmFromFile operator is used, and many functions from lesser-used PHP extensions are removed. To mitigate false positives, we only match when the '(' character is also found.
This rule only runs in paranoia level 2 or higher.

**Comment by fzipi** [comment]
The list of php Zend functions can be obtained from the php source code using:
`ag --no-numbers 'F.\(' Zend/Optimizer/zend_func_infos.h | cut -f2 -d\" | sort`

**Comment by fzipi** [comment]
A more comprehensive list can be obtained using: 
`grep -o --no-file -R 'ZEND_FUNCTION(.*)' | cut -f2 -d\( | cut -f1 -d\) | sort`

**Comment by M4tteoP** [comment]
Closing as completed via https://github.com/coreruleset/coreruleset/pull/3273
